File: orchestrator/android/ExoPlayer-Java/app/src/main/java/io/streamroot/ctl/delivery/client/samples/orchestrator/exoplayer/SRApplication.java
package io.streamroot.ctl.delivery.client.samples.orchestrator.exoplayer;

import androidx.multidex.MultiDexApplication;

import io.streamroot.ctl.delivery.client.core.CTLDeliveryClient;

public final class SRApplication extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        CTLDeliveryClient.initializeApp(this);
    }
}

File: orchestrator/android/ExoPlayer/app/src/main/java/io/streamroot/ctl/delivery/client/samples/orchestrator/exoplayer/SRApplication.kt
package io.streamroot.ctl.delivery.client.samples.orchestrator.exoplayer

import androidx.multidex.MultiDexApplication
import io.streamroot.ctl.delivery.client.core.CTLDeliveryClient

class SRApplication: MultiDexApplication() {

    override fun onCreate() {
        super.onCreate()
        CTLDeliveryClient.initializeApp(this)
    }
}